receiver/elasticapmreceiver/receiver.go

```diff
@@ -104,6 +110,10 @@ func (r *elasticAPMReceiver) startHTTPServer(ctx context.Context, host component
 	}
 
 	r.settings.Logger.Info("Starting HTTP server", zap.String("endpoint", r.cfg.ServerConfig.Endpoint))
+
+	// TODO: Remove this once we have a proper way to configure TLS
+	r.cfg.TLSSetting = nil
```

Comment:
nit: could we set this in the factory.go `createDefaultConfig` default method instead? (to avoid mutating the config after being parsed)